lib/matplotlib/patches.py

```diff
@@ -405,8 +401,6 @@ def set_linewidth(self, w):
         """
         if w is None:
             w = mpl.rcParams['patch.linewidth']
-            if w is None:
```

Comment:
I cannot really see this happening as `mpl.rcParams['patch.linewidth']` should never return None. Or have I missed sometihng?

Reply:
Correct; the validator says that it must be a float.